app/services/mixins/tag_mixin.rb

```diff
@@ -9,8 +9,4 @@ def fq_tag_name(workflow_id)
   def approval_tag(workflow_id)
     { 'tag' => fq_tag_name(workflow_id) }
   end
-
-  def approval_tag_filter
```

Comment:
This was dead code, the escape never worked so I had to pass them as separate params and have Faraday do the encoding of the params.